src/js/app.js

```
@@ -1,1405 +1,41 @@
-  let done=0;
-  for(const o of ORGS){
-    if(o.github){
-      txt.textContent=`${++done}/${ORGS.length}…`;
-      const d=await fetchGH(o.github);if(d)o._gh=d;
-      await new Promise(r=>setTimeout(r,85));
-    }
-  }
-  spin.style.display='none';btn.disabled=false;txt.textContent='✓ Done';fetching=false;
-  applyFilters();updateStats();
-}
+  if(fetching)return;
```

ESLint

[error] 'fetching' is not defined.

(no-undef)